.pre-commit-config.yaml

```diff
@@ -22,3 +22,16 @@ repos:
         language: script
         types: [yaml]
         pass_filenames: true
+      - id: check-go-generation
+        name: check-go-generation
+        entry: dask_kubernetes/operator/go_client/hack/regenerate-code.sh
+        language: script
+        files: ^dask_kubernetes/operator/go_client/pkg/apis/.*\.go$
```

Comment:
Would we get any benefit there? As a user would have to manually keep the go files in sync. So in theory, the code-generation should always be valid in case no go files are touched?

Reply:
Ah I see, I assumed the go files were autogenerated from the CRDs.

Reply:
Sadly not! I looked around a bit and seems like that's not possible at the moment :-(